test/jdk/java/lang/ProcessBuilder/Basic.java

```diff
@@ -2135,10 +2135,33 @@ public void doIt(Map<String,String> environ) {
             final int cases = 4;
             for (int i = 0; i < cases; i++) {
                 final int action = i;
-                List<String> childArgs = new ArrayList<String>(javaChildArgs);
+                List<String> childArgs = new ArrayList<>(javaChildArgs);
+                final ProcessBuilder pb = new ProcessBuilder(childArgs);
+                {
```

Comment:
I was a little confused until I looked up the JavaDoc of ProcessBuilder.

Maybe add a comment like "ProcessBuilder does NOT make a copy of childArgs so it's OK to modify it below"?